config/devel/manager_image_patch.yaml

```diff
@@ -10,3 +10,6 @@ spec:
       - name: devworkspace-controller
         image: ${IMG}
         imagePullPolicy: Always
+        env:
+        - name: RELATED_IMAGE_devworkspace_webhook_server
+          value: ${IMG}
```

Comment:
We had the same idea :) https://github.com/devfile/devworkspace-operator/pull/196